www/class/config-generate-remote/Engine.php

```diff
@@ -31,7 +31,7 @@ class Engine extends AbstractObject
     protected $table = 'cfg_nagios';
     protected $generateFilename = 'cfg_nagios.infile';
 
-    //skipped nagios parameters : temp_file, nagios_user, nagios_group, log_rotation_method, log_archive_path,
+    //skipped nagios parameters : temp_file, nagios_user, nagios_group, log_rotation_method, log_archive_path (why?),
```

Comment:
code is not meant to chat ;)
ask your question using available medias, like slack.